torch_tvm/compiler.h

```diff
@@ -10,6 +10,8 @@
 
 #include <vector>
 
+tvm::relay::DataType scalarTypeToTVMType(at::ScalarType pt_type);
```

Comment:
I don't think we want this public -- is there a good reason?

Reply:
This is needed in operator.cpp and I think we would need it to extract TVM type in other places where right now it is hardcoded to float.